packages/layout/src/components/cards/common/views/remove/date/date.tsx

```diff
@@ -45,14 +44,7 @@ const DateForm: React.FC<DateFormProps> = ({
 							cfg={{ mb: 3 }}
 						/>
 						<div className={styles.dateWrapper}>{renderFields(dateField)}</div>
-						{submitError && (
-							<P
-								cfg={{ color: 'danger.2', mt: 5 }}
-								className={CommonStyles.noMarginBottom}
-							>
-								{submitError}
-							</P>
-						)}
+						{submitError && <P className={styles.errorMsg}>{submitError}</P>}
```

Comment:
Is the accessibility definitely OK here? Normally I'd expect to see an id on an error message, so that it can be linked to the field using aria-describedby.

Reply:
`id` added and linked to `fieldset`